src/main/scala/ch/epfl/bluebrain/nexus/kg/resources/Event.scala

```diff
@@ -108,4 +109,32 @@ object Event {
       identity: Identity
   ) extends Event
 
+  /**
+    * A witness that a resource's attachment has been added.
+    *
+    * @param id       the resource identifier
+    * @param rev      the revision that this event generated
+    * @param value    the metadata of the attachment
+    * @param instant  the instant when this event was recorded
+    * @param identity the identity which generated this event
+    */
+  final case class Attached(id: Id[ProjectRef],
```

Comment:
Could we rename the event types to `AttachmentAdded` and `AttachmentRemoved` to match the `Tag*` naming?

Reply:
Same for commands...